doc.yml

```diff
@@ -0,0 +1,12 @@
+doc:
+  stage: doc
+  image: curlimages/curl
+  script:
+    - curl -L https://github.com/Kerwood/confluence-updater/releases/download/v1.1.1/confluence-updater-x86_64-unknown-linux-musl -o confluence-updater
+    - chmod +x confluence-updater
```

Comment:
I think it will be great to have a custom image (on Docker Hub, @Jobtome) with everything pre-bundled. :))

Reply:
did this in our registry :) just wanted to test that path